mkdocs.yml

```
@@ -1,5 +1,5 @@
-site_name: 'Documentation'
-site_description: 'Wiki about the CI of CodeMC'
+site_name: 'CodeMC Documentation'
+site_description: 'Official Wiki of CodeMC'
```

Maybe a better description?
```suggestion
site_description: 'Official Wiki for the CodeMC Project'
```